### PR TITLE
Add berachain networks

### PR DESCRIPTION
**Pull request by xsubject**
This PR adds the testnet of the new [berachain network](https://twitter.com/berachain?lang=en)

---

## PR-Codex overview
This PR focuses on adding support for the Berachain Testnet chain.

### Detailed summary
- Added Berachain Testnet chain.
- Added `berachainTestnet` definition in `src/chains/definitions/berachainTestnet.ts`.
- Updated `src/chains/index.ts` to export `berachainTestnet` definition.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/chains/definitions/berachainTestnet.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  blockExplorers: {
+    default: {
+      name: 'Berachain',
+      url: 'https://artio.beratrail.io/',
+      apiUrl: 'https://api.routescan.io/v2/network/testnet/evm/80085/etherscan',
+    },
+  },
+  contracts: {
+    multicall3: {
+      address: '0x9d1dB8253105b007DDDE65Ce262f701814B91125',
```
Comment (tmm) [inline src/chains/definitions/berachainTestnet.ts:23]: Only verified contracts are accepted.

File: src/chains/definitions/berachainTestnet.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    name: 'tBera Coin',
+    symbol: 'tBERA',
+  },
+  rpcUrls: {
+    default: { http: ['https://artio.rpc.berachain.com/'] },
+  },
+  blockExplorers: {
+    default: {
+      name: 'Berachain',
+      url: 'https://artio.beratrail.io/',
```
Comment (tmm) [inline src/chains/definitions/berachainTestnet.ts:17]:
```suggestion
      url: 'https://artio.beratrail.io',
```

File: src/chains/definitions/berachainTestnet.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+export const berachainTestnet = /*#__PURE__*/ defineChain({
+  id: 80085,
+  name: 'Berachain Testnet',
+  nativeCurrency: {
+    decimals: 18,
+    name: 'tBera Coin',
+    symbol: 'tBERA',
+  },
+  rpcUrls: {
+    default: { http: ['https://artio.rpc.berachain.com/'] },
```
Comment (tmm) [inline src/chains/definitions/berachainTestnet.ts:12]:
```suggestion
    default: { http: ['https://artio.rpc.berachain.com'] },
```

File: .changeset/nice-fishes-leave.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+"viem": patch
+---
+
+Add berachain testnet
```
Comment (tmm) [inline .changeset/nice-fishes-leave.md:5]:
```suggestion
Added Berachain Testnet chain.
```

**Review by tmm — CHANGES_REQUESTED** [review]

**Review by jxom — COMMENTED** [review]
Can you please get this chain listed here? https://github.com/ethereum-lists/chains

I can't seem to find it

**Comment by codingwithmanny** [comment]
@jxom do you happen to know some of the maintainers? We have a PR open, but just waiting on it to get merged. https://github.com/ethereum-lists/chains/pull/4158

File: src/chains/definitions/berachainTestnet.ts (hunk 1 above)
Comment (codingwithmanny) [reply src/chains/definitions/berachainTestnet.ts:23]: Will get that for you @tmm

**Comment by xsubject** [comment]
@jxom I think it's ready, please check it now

File: src/chains/definitions/berachainTestnet.ts (hunk 1 above)
Comment (jxom) [reply src/chains/definitions/berachainTestnet.ts]: Removed for now. Feel free to open another PR once verified.

**Pull request merged**